src/containers/course-section/CourseSectionContainer.tsx

```diff
@@ -46,20 +46,18 @@ import {
 } from '~/types'
 import { styles } from '~/containers/course-section/CourseSectionContainer.styles'
 
+type CourseSectionChangeHandler<TValue> = (
```

Comment:
move to types, in components we define only interfaces for props

Reply:
Sounds great! I type intead, because interface cannot be used to type function and moved in to common.types